lib/blackbeard/version.rb

```diff
@@ -1,3 +1,3 @@
 module Blackbeard
-  VERSION = "0.0.5.3"
+  VERSION = "0.0.6.0"
```

Comment:
leave version as is. I will bump when I release gem.  In meantime, reference by commit in Gemfile.